info.yml

```diff
@@ -41,6 +41,18 @@ configuration:
         default_value: 5
         description: Number of versions to display in the file history information window.
 
+    publish_history_group_by_fields:
+        type: list
+        description: The Published File fields to use when looking for the publish history. They are
+                     used to match the current Published File data against other Published Files. By default,
+                     the app will look for Published Files with the same project, task, entity, name and
```

Comment:
Please remove the "By default" comment, or add some code to do that if the setting is empty.